packages/shared/src/components/cards/ShowMoreContent.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { ReactElement, ReactNode, useState } from 'react';
+import { ClickableText } from '../buttons/ClickableText';
+
+interface ShowMoreContentProps {
+  content: string;
+  charactersLimit?: number;
+  threshold?: number;
+  contentPrefix?: ReactNode;
+  className?: string;
+}
+
+export default function ShowMoreContent({
+  content,
+  charactersLimit = 150,
+  threshold = 50,
+  contentPrefix,
+  className,
+}: ShowMoreContentProps): ReactElement {
+  const [isTextExpanded, setIsTextExpanded] = useState(false);
+  const linkName = isTextExpanded ? 'Show less' : 'Show more';
+
+  const toggleTextExpanded = () => {
+    setIsTextExpanded(!isTextExpanded);
+  };
+
+  const displayShowMoreLink = () => {
+    if (content && content?.length > charactersLimit + threshold) {
+      return true;
+    }
+    return false;
+  };
+
+  const getContent = () => {
+    return isTextExpanded ? content : content?.slice(0, charactersLimit);
+  };
+  return (
+    <div className={className}>
+      <p className="typo-body">
+        {contentPrefix}
+        {getContent()}{' '}
+        {displayShowMoreLink() && (
+          <ClickableText
+            className="inline-flex text-theme-label-link"
+            onClick={toggleTextExpanded}
+            tag="a"
```

Comment:
Why is that a `a` element? I think the `tag` property can be dropped

Reply:
I'm pretty sure this should render as a <a> not a <button> right?
Since it's an inline anchor element.

The button would also push it to it's own line.

Reply:
Hmm interesting. The second argument could be easily fixed but the first one is interesting.
I think anchors should only be links and this one is actually a button. So in terms of accessibility and semantics, I do think it's a button but you made me somewhat question it. Could you check it, please?

Reply:
![Screenshot 2022-01-20 at 13 42 28](https://user-images.githubusercontent.com/554874/150332335-14dae880-108a-4e05-8b2c-ab13b54957c1.png)

To me this is really a <a href=""> not a <button>

But let me know if you think otherwise

Reply:
But according to A11Y rules you are 100% right and it should be a button (my bad)
![Screenshot 2022-01-20 at 13 44 34](https://user-images.githubusercontent.com/554874/150332691-cd233267-f7d1-4a92-b84f-83377f40e546.png)

Reply:
Reading a bit more, there seems to be quite the split thinking about these and some field experts even say it should not even exist in this way.

https://blog.prototypr.io/youre-not-still-using-read-more-are-you-16466ae326e7

Reply:
Let's go with a button then :)